docs/en/operations/settings/settings.md

```diff
@@ -260,6 +260,39 @@ If an error occurred while reading rows but the error counter is still less than
 
 If both `input_format_allow_errors_num` and `input_format_allow_errors_ratio` are exceeded, ClickHouse throws an exception.
 
+## input_format_parquet_import_nested {#input_format_parquet_import_nested}
+
+Enables or disables the ability to insert `Array(Struct)` values into [Nested](../../sql-reference/data-types/nested-data-structures/nested.md) columns in [Parquet](../../interfaces/formats.md#data-format-parquet) input format.
```

Comment:
Let's change here and below too:

"Enables or disables the ability to insert data into [Nested](../../sql-reference/data-types/nested-data-structures/nested.md) columns as an array of structs in ..."

and 

"0 — Data can not be inserted into `Nested` columns as an array of structs.
-   1 —Data can be inserted into `Nested` columns as an array of structs."

And in Russian version